src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.VisualBasic.targets

```diff
@@ -10,6 +10,30 @@ Copyright (c) .NET Foundation. All rights reserved.
 ***********************************************************************************************
 -->
 <Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <ItemGroup Condition=" '$(DisableImplicitNamespaceImports)' != 'true' and '$(TargetFrameworkIdentifier)' == '.NETFramework'">
```

Comment:
May I please get a clarification why VB's implicit imports are now hidden behind `DisableImplicitNamespaceImports`, which is `on` by default? VB always had implicit imports on by default AFAIK.

This has broken https://github.com/dotnet/winforms/pull/5437

/cc: @KathleenDollard @KlausLoeffelmann @DamianEdwards

Reply:
IIUIC, https://github.com/dotnet/sdk/pull/18459/files#diff-5dfdd31992820273a25b399353a85ec6de6bdd620424a8adab3238c6ffd5df38 has removed a condition which hasn't been restored:
![image](https://user-images.githubusercontent.com/4403806/129832674-88f21d72-5f98-4a04-9993-2a6b281c4041.png)

Reply:
I think this may be [the culprit](https://github.com/dotnet/sdk/pull/18459/files#diff-49cf43685a82b532902ba9b23390b53738fe901f67664b7a1aae71c0444e0dec), which needs to be undone:
![image](https://user-images.githubusercontent.com/4403806/129834463-eabfd0e2-4d65-49e4-ad92-6dca083f59f8.png)

Reply:
For a test I built a .NET 5.0 VB app, and neither `DisableImplicitNamespaceImports` nor `DisableImplicitFrameworkReferences` are defined.

E.g.
```
dotnet new winforms --language vb -f net5.0
dotnet build /bl
start msbuild.binlog
```

![image](https://user-images.githubusercontent.com/4403806/129835594-dd26fd4e-67da-4af4-a61c-d663cacb7d4b.png)
![image](https://user-images.githubusercontent.com/4403806/129835577-01aadb84-8dc1-4270-9e28-c3f836cbd8c9.png)

And it has all imports:
![image](https://user-images.githubusercontent.com/4403806/129835938-8cc37525-3e86-4a86-915a-4e80de581a64.png)

Reply:
Looks like this might have been missed. @pranavkm?

Reply:
Why does the screen shot show "net6.0-windows" if this is a " .NET 5.0 VB app"?

Reply:
The issue that @RussKie is running in to is because of an Arcade change - https://github.com/dotnet/arcade/pull/7745. That said, I missed undoing one of the VB changes that is covered as part of https://github.com/dotnet/sdk/pull/19840